Directory.Build.props

```diff
@@ -312,6 +319,15 @@
     <IncludeDllSafeSearchPathAttribute Condition="'$(TargetGroup)' == 'netstandard1.0'">false</IncludeDllSafeSearchPathAttribute>
   </PropertyGroup>
 
+  <!-- Set up default paths for reference assemblies -->
+  <PropertyGroup Condition="'$(IsReferenceAssembly)'=='true'">
+    <!-- Create a common root output directory for all reference assemblies -->
+    <ReferenceAssemblyOutputPath Condition="'$(ReferenceAssemblyOutputPath)' == ''">$(BaseOutputPath)ref/</ReferenceAssemblyOutputPath>
```

Comment:
Why did these get moved here instead of using the ones in BuildTools? https://github.com/dotnet/buildtools/blob/16a6d1f39cf0a572041d51bd318913548ba41b34/src/Microsoft.DotNet.Build.Tasks/PackageFiles/ReferenceAssemblies.targets#L15

Reply:
I’m not entirely sure why this changed. I asked @wtgodbe and he said that change was included with @joperezr’s initial changes. I can try reverting it to see what happens.

Reply:
I'd like to understand if it is needed here still.